courses/free-courses-en.md

```diff
@@ -216,6 +216,7 @@
 * [Firebase in a Weekend: Android](https://www.udacity.com/course/firebase-in-a-weekend-by-google-android--ud0352) (Udacity)
 * [Gradle for Android and Java](https://www.udacity.com/course/gradle-for-android-and-java--ud867) (Udacity)
 * [Jetpack Compose](https://www.youtube.com/playlist?list=PLQkwcJG4YTCSpJ2NLhDTHhi6XBNfk9WiC) - Philipp Lackner
+* [Kotlin & Android 12 Tutorial | Learn How to Build an Android App](https://youtu.be/HwoxgUPabMk?si=rPuiEeylvAwgf7Eg) - tutorialsEU
```

Comment:
no shortened links
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes